custom/enikshay/ucr_filters.py

```diff
@@ -24,6 +24,13 @@ def get_hierarchy(self, location_id):
         """
         relevant_location_types = ["sto", "cto", "dto", "tu", "phi"]
         location = SQLLocation.objects.get(location_id=location_id)
+
+        if location.location_type.name not in relevant_location_types:
```

Comment:
relevant_location_types look like thye might be site codes and not names. is name the correct thing here?